### DISCUSSION TITLE
SASS-like features

### DISCUSSION THREAD

**Discussion by jeromecovington** [postcss/postcss#1796] [Q&A] [answered]
If I wanted near-parity with SASS, what plugins would I enable in my postcss.config?

**Comment by ai** [comment, marked as answer]
We do not recommend to make PostCSS to Sass. We promote avoid complex logic in CSS and have more declarative styles (instead of making programs in styles like in Sass). We also promote CSSWG specs instead of custom syntax.

I recommend using `postcss-preset-env` plugin with CSS Custom Properties and CSS native nesting.
